utils/docker/Dockerfile.centos.7

```diff
@@ -173,14 +173,14 @@ RUN [ "$DAOS_DEPS_BUILD" != "yes" ] || \
 # force an upgrade to get any newly built RPMs
 USER root:root
 ARG CACHEBUST
-RUN dnf -y upgrade \
-        --exclude=spdk,spdk-devel,dpdk-devel,dpdk,mercury-devel,mercury && \
-    dnf clean all
+RUN [ -z "$CACHEBUST" ] || \
```

Comment:
In Jenkins it it yes, but in GHA it's unset, so this will turn off the upgrade in that setting.  In practice it was skipped anyway unless there was a deps change so this only affects PRs which change the dependency builds but not the dockerfiles/package lists.  Turning this off allows faster builds on all targets, and removes a potential source of failure.

Reply:
I would argue that it could introduce failure as builds are done with out-of-date packages.

Reply:
Potentially out-of-date, but known good, in fact the most recent version to have worked.